backend/curriculum_tracking/serializers.py

```diff
@@ -355,8 +367,21 @@ class Meta:
             "open_pr_count",
             "oldest_open_pr_updated_time",
             "repo_url",
+            "number_of_times_card_moved_to_review_feedback",
         ]
 
+    number_of_times_card_moved_to_review_feedback = serializers.SerializerMethodField("get_number_of_times_card_moved_to_review_feedback")
+
+    def get_number_of_times_card_moved_to_review_feedback(self, instance):
+        if instance.recruit_project:
+            number_of_times_card_moved_to_review_feedback = len(
+                LogEntry.objects.filter(effected_user=instance.assignees.first(), 
+                object_1_id=instance.recruit_project.pk, 
```

Comment:
This query should be more explicit. 

We can have a recruit project with the same id as a repo. We can have a recruit project with the same id as a review as well.

Please also add object_1's content type to the query. Explicit is better than implicit.